huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_user.go

```diff
@@ -172,7 +173,7 @@ func resourceDmsRocketMQUserCreate(ctx context.Context, d *schema.ResourceData,
 	return resourceDmsRocketMQUserRead(ctx, d, meta)
 }
 
-func buildCreateRocketmqUserBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateRocketmqUserBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```

Comment:
If the config is not used, we should remove it from the arguemnt list.

Reply:
done